nservicebus/hosting/docker-host/index.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Docker Container Host
+summary: Take advantage of process isolation by hosting endpoints in Docker containers
+tags:
+ - Hosting
+related:
+ - nservicebus/dotnet-templates
+ - nservicebus/licensing
+ - samples/hosting/docker
+component: Templates
+versions: '[2,]'
+reviewed: 2018-02-28
+---
+
+Docker containers provide the ability to deploy endpoints in a self-contained manner. To create an endpoint to host in a Docker container, install the [Docker endpoint template](/nservicebus/dotnet-templates.md) and create a new project using `dotnet new nsbdockerendpoint`. The project that is created will have the required endpoint setup infrastructure in addition to the needed `Dockerfile` to create and deploy a container hosting one endpoint.
```

Comment:
The name of the template needs to be updated here, along with the `dotnet new` command.